### ISSUE TITLE
[BUG] Accessibility - Web app - "Skip to" link missing

### ISSUE THREAD

**Issue by acovher2** [rokwire/illinois-app#5143]
**Describe the bug**
“Skip to” link is missing. This is a crucial feature for web accessibility. This is a hyperlink placed at the top of a webpage that allows users to bypass repetitive navigation links and jump straight to the main content. Screen reader users often navigate through many links and menu items before reaching the main content. A “skip to” link allows them to bypass this repetitive content and save time and effort. This also helps keyboard users as they can quickly get to main content. For users with motor impairments, reducing the number of keystrokes needed to reach the main content can make the web experience significantly more manageable and less tiring.

**To Reproduce**
1. Go to app.illinois.edu
2. Begin using TAB to navigate, notice there is no "Skip to" link,

**Expected behavior**
There should be a "Skip to" link which allows the user to bypass regular navigation items and skip to main content

**Actual behavior**
There is no skip to link. 

**Screenshots**

**Desktop (please complete the following information):**
PC:
•	OS: Windows 11
•	Device: Dell laptop
•	Browser: Chrome 135.0.7049.116/Firefox 138.0.1

**Additional context**
